nl/xfrm_state_linux_test.go

```diff
@@ -205,3 +205,12 @@ func TestXfrmEncapTmplDeserializeSerialize(t *testing.T) {
 	msg := DeserializeXfrmEncapTmpl(orig)
 	testDeserializeSerialize(t, orig, safemsg, msg)
 }
+
+func TestSerializeDeserializeXfrmMark(t *testing.T) {
```

Comment:
Any reason you didn't follow the format of all the other DeserializeSerialize tests?